spec/helpers/session.rb

```diff
@@ -0,0 +1,22 @@
+module SessionHelpers
+
+  def sign_up(name: 'New User', username: 'whoami', email: 'user@email.com', password: 'password', password_confirm: 'password')
```

Comment:
Metrics/LineLength: Line is too long. [128/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)